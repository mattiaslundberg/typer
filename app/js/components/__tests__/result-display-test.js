import React from 'react'
import ResultDisplay from '../result-display'
import renderer from 'react-test-renderer'

describe('ResultDisplayTest', () => {
  const render = (time, chars) => { return renderer.create(
    <ResultDisplay time={time} textLength={chars} />
  ) }

  it('WPM calculation is correct', () => {
    let component = render(10, 50)

    expect(component.toJSON()).toMatchSnapshot()
  })
})
