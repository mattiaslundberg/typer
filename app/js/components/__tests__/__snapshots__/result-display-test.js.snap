exports[`ResultDisplayTest WPM calculation is correct 1`] = `
<div>
  <div>
    Time: 
    0
    s
  </div>
  <div>
    Chars/m: 
    300000
  </div>
  <div>
    Words/m: 
    60000
  </div>
  <button
    onClick={undefined}>
    Try again
  </button>
</div>
`;
